episodes/18-preparing.md

```diff
@@ -48,27 +47,57 @@ but reduces a lot the preparation time of future lessons and are super useful to
 
 \- Dr. Allegra Via, Carpentries Instructor and Instructor Trainer
 
+::::::::::::::::::::::::::::::::::::::::::::::::::
+
+
+::::::::::::::::::::::::::::::::::::::  challenge
+
+## Exercise: Prepare a teaching plan (15 minutes)
+
+Create a bullet point list or brief notes describing
+what you will say and do when teaching the section you have been focussing on
+during this training.
 
 ::::::::::::::::::::::::::::::::::::::::::::::::::
 
-- Using figures and visual aids
-  - Remember the alt text
-  - Call back to placing explanatory text adjacent to relevant part of visualisation
-- Working with the lesson infrastructure:
-  - Setup instructions
-    - This is time well-invested. Clear instructions now may give you more time when teaching the lesson.
-  - Discuss briefly what belongs in Instructor Notes pages:
-    - strengths and weaknesses of current design
-    - what has been tried before and what worked/did not work
-    - tips for teaching the lesson/particular sections
-    - challenges encountered
-    - parts that seem to cause biggest trouble/most confusion for learners (i.e. where to tread carefully), etc
-- Collecting feedback
-  - a reminder of some of the methods for collecting feedback that can be most helpful for a trial run
-    - minute cards
-    - one up, one down
-    - perhaps even some more specific feedback, for a relatively short trial run e.g. how clear were the exercises? were they appropriately challenging i.e. learner felt like they had to think about it to complete the challenge, but not so difficult that they gave up/became frustrated.
-  - link to relevant section of Instructor Training
+Figures and other visual aids are great tools to help participants understand the content being presented and make associations between pieces of information. If you are planning on using these in your lesson or teaching session - remember to enhance them with the ALT text (invisible description of images which are read aloud to blind users on screen readers providing alternative text-based format for images) or place explanatory text adjacent to relevant part of visualisation.
```

Comment:
The sentence with the description of alt text in it is a bit long.  I suggest removing that description and make sure it is well covered in the explanation episode with a shorter reminder here.
```suggestion
Figures and other visual aids are great tools to help participants understand the content being presented and make associations between pieces of information. If you are planning on using these in your lesson or teaching session - remember to enhance them with the ALT text describing the figure for individuals using screen readers or by placing explanatory text adjacent to relevant part of visualisation.
```

Reply:
I have accepted your changes @sstevens2 just was not sure which one is "the explanation episode" where you suggested we covered this in detail. Do we need anything more at all? I like how it reads now.

Reply:
This is good. The section Sarah referred to is in _How to write a lesson_. It includes a (currently optional) exercise looking at alt-text for data visualisations.

I like how it is written after this change, and I think this is ready to go.